src/NUnitEngine/nunit.engine/Communication/Transports/Tcp/TcpServer.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+
+using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading;
+using NUnit.Engine.Internal;
+
+namespace NUnit.Engine.Communication.Transports.Tcp
+{
+    public class TcpServer
+    {
+        private static readonly Logger log = InternalTrace.GetLogger(typeof(TestAgentTcpTransport));
```

Comment:
```suggestion
        private static readonly Logger log = InternalTrace.GetLogger(typeof(TcpServer));
```

Presumably should be this?